### ISSUE TITLE
notched boxplots

### ISSUE THREAD

**Issue by grero** [JuliaPlots/StatsMakie.jl#59]
I'm a bit confused about the way notches on box plots are currently rendered. This is how I'm used to seeing them: [boxplots](https://en.wikipedia.org/wiki/Box_plot#/media/File:Fourboxplots.svg)
However, this is how StatsMakie renders them.
```julia
using StatsMakie
using Random
scene = boxplot(fill(1.0, 100), rand(100),notch=true
```
![statsmakie_boxplot](https://user-images.githubusercontent.com/1245900/51222364-630bd300-1978-11e9-9ec8-310f91401e16.png)

For reference, this is how it looks when I plot using PyPlot
```julia
using PyPlot
using Random
boxplot(rand(100),notch=true)
```
![pyplot_boxplot](https://user-images.githubusercontent.com/1245900/51222403-8afb3680-1978-11e9-860c-30bd8445dde3.png)

**Comment by piever** [comment]
Thanks for reporting, it's definitely a bug.

**Comment by SimonDanisch** [comment]
Interestingly, this works in Plots + gr, which I believe this recipe got ported from. Not sure if the port is buggy, or if the original recipe got improved!
At least it only seems to be a corner case, since "normal" box plots still seem to work fine

**Comment by grero** [comment]
Yes, the code works beautifully for "normal" boxplots : )

**Comment by piever** [comment]
Fixed by #64
